src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusTopicMessagePumpOptions.cs

```diff
@@ -5,17 +5,10 @@ namespace Arcus.Messaging.Pumps.ServiceBus.Configuration
     /// <summary>
     ///     Options to configure a Azure Service Bus topic <see cref="AzureServiceBusMessagePump"/> implementation.
     /// </summary>
+    /// TODO: remove 'old' workings after the background jobs package is updated with the new messaging package.
+    [Obsolete("Will be removed in the future, please use the " + nameof(AzureServiceBusMessagePumpOptions) + " instead")]
     public class AzureServiceBusTopicMessagePumpOptions : AzureServiceBusMessagePumpOptions
```

Comment:
This is a 1.0 release, please just remove it

Reply:
Removed, resolving.